.readthedocs.yml

```diff
@@ -1,8 +1,9 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 sphinx:
-  configuration: docs/conf.py
+  configuration: components/google-cloud/docs/source/conf.py
```

Comment:
This seems to be an unintended change?

Reply:
This is to temporarily work around this issue: https://github.com/kubeflow/pipelines/pull/5887/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9L4

Reply:
I see. Thanks for the pointer.